# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xemacs-beta/nostdlib.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual linking hacks.

  - Rene Rebe <rene@exactcode.de>

diff -ur xemacs-21.5.17/src/s/linux.h xemacs-21.5.17-xft/src/s/linux.h
--- xemacs-21.5.17/src/s/linux.h	2002-08-12 16:45:34.000000000 +0200
+++ xemacs-21.5.17-xft/src/s/linux.h	2004-04-27 05:14:01.000000000 +0200
@@ -116,7 +116,7 @@
 #else
 #define LIB_STANDARD "-lgcc -lc -lgcc /usr/lib/crtn.o"
 #endif
-#define LINKER "$(CC) -nostdlib"
+#define LINKER "$(CC)"
 #endif
 
 #ifdef TERM
